Add troubleshooting entry for combined index privilege patterns (#7311)
elastic/docs-content: troubleshoot/security/detection-rules.md

::::

## Rule privileges [rule-privileges-ts]

::::{dropdown} Warning about required read privileges for indices you can access
:name: rule-privileges-combined-patterns-ts

> A rule might warn that it may not have the required read privileges for index patterns that your role clearly includes, such as:

Reviewer: ```suggestion
You might receive the following warning for a rule even when your role clearly includes privileges for the index patterns:
```
Not sure if this works completely, but the "might" and "may" were messing me up when trying to understand. Up to your discretion, of course!

Author: Thanks. This is better,
